client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -202,6 +206,20 @@ static StreamManager create(ClientConfig clientConfig) {
      */
     boolean deleteScopeRecursive(String scopeName) throws DeleteScopeFailedException;
 
+    /**
+     * Re-read an event that was previously read, by passing the pointer returned from
+     * {@link EventRead#getEventPointer()}.
+     * This does not affect the current position of the reader.
+     * <p>
+     * This is a non-blocking call. Passing invalid offsets will throw exception.
+     * @param pointer The pointer object to enable a random read of the event.
```

Comment:
Please clarify this. Specifically it is an eventPointer obtained from the result of a previous readNextEvent call